backend/app/services/rag/storage/elasticsearch_backend.py

```
@@ -0,0 +1,505 @@
+                    ]
+                }
+            },
+            "sort": [{"metadata.chunk_index": "asc"}],
+            "_source": ["content", "metadata"],
+        }
+
+        response = es_client.search(index=index_name, body=search_body)
+
+        hits = response["hits"]["hits"]
+        if not hits:
+            raise ValueError(f"Document {document_id} not found")
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)